pkg/prometheus/operator.go

```diff
@@ -352,9 +354,19 @@ func (c *Operator) RegisterMetrics(r prometheus.Registerer, reconcileErrorsCount
 		Name: "prometheus_operator_node_address_lookup_errors_total",
 		Help: "Number of times a node IP address could not be determined",
 	})
+	c.nodeEndpointSyncs = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_operator_node_syncs_total",
+		Help: "Number of times node endpoints have been synchronized",
+	})
+	c.nodeEndpointSyncErrors = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_operator_node_syncs_failed_total",
+		Help: "Number of times node endpoints couldn't be synchronized",
```

Comment:
Nit, wdyt of this for the help text: "Number of node endpoints synchronisation failures."